brion/morphology.h

```diff
@@ -37,7 +37,18 @@ class Morphology
     /** Close morphology file. @version 1.0 */
     BRION_API ~Morphology();
 
-    /** @name Read API */
+    BRION_API Morphology(Morphology&&);
+    BRION_API Morphology& operator=(Morphology&&);
+
+    /**
+     * @name Read API
+     *
+     * Data returned by the read methods may or may not be cached by the
+     * implementation. If the returned vectors are modified this may be
+     * reflected in subsequent reads of the same data. Furthermore, calling read
+     * multiple times might be expensive. It is therefore recommended to call
+     * each read method at most once per instantiation.
```

Comment:
Out of the scope for this PR. Imo the motivation is to keep Brion simple by placing as little constraints on the impl as possible. All existing users follow the recommendation anyways (call read once). I tried to got the const route, but this has it's own can of worms which smelled fishy (mutable on unique_ptr, different behaviour if called twice, or memcpy).